phpunit.xml

```diff
@@ -10,7 +10,10 @@
         stopOnFailure="false">
     <testsuites>
         <testsuite name="Application Test Suite">
```

Comment:
If we are calling each testSuite individually, there should not be a general one that does them both

Reply:
u right!